lib/CLI/Types.hs

```diff
@@ -56,6 +56,7 @@ data CopyResult
   | CPRPathNotFound Path
   | CPRMissingEntryName
   | CPRDestinationIsDirectory [(EntryPath, EntryPath)]
+  | CPRDestinationHasEntry EntryPath
```

Comment:
How about renaming `CPRDestinationHasEntry` to `CPRParentDirectoryIsEntry`?

To me, the name "destination has entry" implies that we found an entry at the destination path. E.g., we tried to copy `a/b/c` to `e/f/g/`, but an entry already exists at `e/f/g`. But that's not what this error means.

"Parent directory is entry" is a better fit, in my opinion.

Reply:
Done